pkgs/development/libraries/unixODBCDrivers/default.nix

```
@@ -174,4 +174,45 @@
       maintainers = with maintainers; [ spencerjanssen ];
     };
   };
+
+  redshift = stdenv.mkDerivation rec {
+    pname = "redshift-odbc";
+    version = "1.4.45.1000-1";
+
+    src = fetchurl {
+      url = "https://s3.amazonaws.com/redshift-downloads/drivers/odbc/1.4.45.1000/AmazonRedshiftODBC-64-bit-${version}.x86_64.deb";
+      sha256 = "sha256-WCntyp+Him8huzTlIE/tPwnzBj6zyw7A3S8mz1gqeTU=";
+    };
+
+    unpackPhase = ''
+      ${dpkg}/bin/dpkg -x $src src
```

```suggestion
      dpkg -x $src src
```
Please add it to nativeBuildInputs